packages/platforms/delta/x86-64/x86-64-delta-ag9032v1/onlp/builds/src/module/src/platform_lib.c

```diff
@@ -0,0 +1,250 @@
+/************************************************************
```

Comment:
There are different numbers and types of CPLD in our platform and it  might be difficult to share the plaform_lib for all delta platform